hips/hip-0012.md

```diff
@@ -126,26 +296,28 @@ In this case, the Helm maintainers may hold a "circuit breaker" vote to halt Hel
 
 The process for halting Helm 4 development will be a simple majority vote by the Helm project maintainers.
 
-#### Maintaining Helm 3
+### Maintaining Helm 3
 
 This section outlines how we will support Helm 3 during Helm 4 development. However, the Release Engineer is at liberty to refine this section in the name of maintaining the best experience for Helm users.
 
 During the "Feature Development" cycle of Helm 4, Helm 3 will continue to receive security, bug, and minor feature patches.
 
 During the Alpha, Beta, and RC phases of Helm 4, Helm 3 will not receive any feature patches. This is to prevent Helm 3 from introducing features that should be in Helm 4, but during a window when Helm 4's features are frozen.
 
-Helm 3 will receive 6 months of bug fixes and one year of security fixes from the day that Helm 4 is released.
+Helm 3 will receive 6-8 months of bug fixes and one year of security fixes from the day that Helm 4 is released.
 
 ## Backwards compatibility
 
 Nothing in this document is contrary to our existing process for decision making.
+Specifically [HIP-0004](./hip-0004.md) largely applies to minor/patch releases only.
+And not a major version release.
 Note that this does not change the PR review process or the HIP process.
 It assumes that both of those processes will remain the same.
 
 ## How to teach this
 
 The contents of this HIP will be discussed in the public Helm Dev meeting, which is recorded.
-If possible, the kick-off meeting will be held at KubeCon North America in 2021,
+If possible, the kick-off meeting will be held at KubeCon North America in 2024,
 and the resources of CNCF will be utilized for advertising this meeting and its purposes.
```

Comment:
In the past, we have had specific meeting space at kubecons outside of session. This time we are doing booth time, contribfest time, and a session. We don't have the space reserved or, I think, have the bandwidth for a separate kickoff meeting. How would you see this unfolding?

Reply:
This is the old verbiage. I corrected the date, without thinking too much about the rest. I'll update the rest shortly